app/views/content/blog/getting-ready-to-apply.md

```diff
@@ -39,7 +39,9 @@ Applications for teacher training courses starting in 2022 open on 12 October, b
 
```

Comment:
Are we updating this blog post date to be 7 September 2022 (why wouldn't we?)

If so, we need to change line 38 re application start date
ie "Applications for teacher training courses starting in 2023 open on 11 October"